internal/validation/validation.go

```diff
@@ -365,14 +362,5 @@ func ValidateUser(typesys *typesystem.TypeSystem, user string) error {
 		}
 	}
 
-	// if the model is a 1.1 model we make sure that the objectType of the 'user' field is a defined
-	// type in the model.
-	if typesystem.IsSchemaVersionSupported(schemaVersion) {
-		_, ok := typesys.GetTypeDefinition(userObjectType)
-		if !ok {
-			return &tuple.TypeNotFoundError{TypeName: userObjectType}
-		}
-	}
```

Comment:
Moved above for easier read.

Reply:
And remove the duplication of checking schema support.